src/PlaywrightSharp.Tests/Page/GoBackTests.cs

```diff
@@ -25,11 +27,11 @@ public async Task ShouldWork()
 
             var response = await Page.GoBackAsync();
             Assert.True(response.Ok);
-            Assert.Equal(TestConstants.EmptyPage, response.Url);
+            Assert.Contains(TestConstants.EmptyPage, response.Url);

```

Comment:
Why contains?

Reply:
this is the correct assert based on playwright